airbyte-webapp/src/components/JobItem/components/MainInfo.tsx

```diff
@@ -64,19 +95,19 @@ const MainInfo: React.FC<MainInfoProps> = ({ job, attempts = [], isOpen, onExpan
       <Cell className={styles.titleCell}>
         <div className={styles.statusIcon}>{statusIcon()}</div>
         <div className={styles.justification}>
-          {isPartialSuccess ? (
-            <FormattedMessage id="sources.partialSuccess" />
-          ) : (
-            <FormattedMessage id={`sources.${getJobStatus(job)}`} />
-          )}
+          {label()}
           {attempts.length > 0 && (
             <>
               {attempts.length > 1 && (
                 <div className={styles.lastAttempt}>
                   <FormattedMessage id="sources.lastAttempt" />
                 </div>
               )}
-              <AttemptDetails attempt={attempts[attempts.length - 1]} configType={getJobConfig(job)} />
+              {jobConfigType === "reset_connection" ? (
+                <ResetStreamsDetails isOpen={isOpen} names={streamsToReset?.map((stream) => stream.name) || []} />
```

Comment:
```suggestion
                <ResetStreamsDetails isOpen={isOpen} names={streamsToReset?.map((stream) => stream.name)} />
```
`ResetStreamsDetails` should probably have a `names = []`.
`export const ResetStreamsDetails: React.FC<ResetStreamsDetailsProps> = ({ names = [], isOpen }) => (`
We don't want to inline create a new object like this.